unittests/finalizer_tests.cpp

```diff
@@ -281,7 +282,7 @@ BOOST_AUTO_TEST_CASE( finalizer_safety_file_versioning ) try {
    // --------------------------------------------------------------------
    fc::temp_directory tempdir;
 
-   for (size_t i=0; i<current_version; ++i) {
+   for (size_t i=1; i<current_version; ++i) {
```

Comment:
Doesn't this now do nothing since `current_version` is 1?

Reply:
Okay. So then should we start the `current_version` at 0 now and change the starting `i` back to 0?

Reply:
done.